src/rdata.cpp

```diff
@@ -809,6 +809,7 @@ void ReturnData::fFIM(int it, Model &model, const ExpData &edata) {
     for (int iy = 0; iy < nytrue; ++iy) {
         if (!edata.isSetObservedData(it, iy))
             continue;
+        int iyt = iy + it * ny;
```

Comment:
just want to note this is not used anywhere.

Reply:
yes, that's part of the code from my old implementation of this, which is a while ago...
will remove, thanks